include/nighthawk/distributor/nighthawk_distributor_client.h

```diff
@@ -0,0 +1,31 @@
+#pragma once
+#include "envoy/common/pure.h"
+
+#include "external/envoy/source/common/common/statusor.h"
+#include "external/envoy/source/common/protobuf/protobuf.h"
+
+#include "api/distributor/distributor.grpc.pb.h"
+
+namespace Nighthawk {
+
+/**
+ * Interface of a gRPC distributor service client.
+ */
+class NighthawkDistributorClient {
+public:
+  virtual ~NighthawkDistributorClient() = default;
+
+  /**
+   * @brief Propagate messages to one or more other services for handling.
+   *
+   * @param nighthawk_distributor_stub Used to open a channel to the distributor service.
+   * @param distributed_request Provide the message that the distributor service should propagate.
+   * @return absl::StatusOr<::nighthawk::DistributedResponse> Either a status indicating failure, or
```

Comment:
nit/optional: can we add a line break between the `@params` and `@return`? Ignore if this is unusual in nighthawk

Reply:
Leaving this as-is, because I checked, and we don't tend to do that in this repository.